package.json

```diff
@@ -15,7 +15,7 @@
     "start": "webpack-dev-server --hot --config webpack.dev.js",
     "storybook": "storybook dev -p 5000",
     "storybook:build": "storybook build",
-    "storybook:deploy": "storybook-to-ghpages",
+    "storybook:deploy": "npx -p @storybook/storybook-deployer storybook-to-ghpages -e storybook-static",
```

Comment:
With this approach, is there a risk that a future version of storybook-deployer could have a breaking change that busts this script?

Reply:
Good question, turns out we won't have to worry about it as apparently storybook-deployer is now deprecated 🤷 
I'll need to investigate a replacement but this should do for now.